docs/source/onboarding_resources/tutorial/index.ipynb

```diff
@@ -0,0 +1,1653 @@
+{
```

Comment:
<blockquote>&nbsp;GBD cause is a&nbsp;<em>disease</em>, such as lung cancer, that causes burden (hence the name)</blockquote><p> </p><p>Actually, a cause in GBD can be either a <em>disease</em> or an <em>injury</em>. If it were just diseases, they probably could have just used the word "disease".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ihmeuw/vivarium_research/pull/1579/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
Ah true, the classic GBD Edge Case™